argyrogounari/assignment1/SinglyLinkedList.java

```diff
@@ -0,0 +1,48 @@
+
+public class SinglyLinkedList<T> {
+
+	private Node<T> head;
+	private Node<T> tail;
+	private int size = 0;
+	
+	public SinglyLinkedList() {};
+	
+	public void findK(int k) {
+		
+		if (k>this.size-1){
+			System.out.println("You can onlu choose a k between 0 and "+ (this.size-1) + ".");
+		} else {
+			Node<T> answer = this.head;
+			for(int i = this.size-1; k<i;k++){
+				answer = answer.getNext();
+				System.out.println("Answer: "+ answer.getData());
```

Comment:
You are printing out Answer: many times. I don't think you need this line as you print the answer below.